frame/elections-phragmen/src/lib.rs

```diff
@@ -1095,17 +1105,18 @@ impl<T: Config> Pallet<T> {
 
 					Self::deposit_event(Event::NewTerm { new_members: new_members_sorted_by_id });
 					<ElectionRounds<T>>::mutate(|v| *v += 1);
-				})
-				.map_err(|e| {
-					log::error!(
-						target: "runtime::elections-phragmen",
-						"Failed to run election [{:?}].",
-						e,
-					);
-					Self::deposit_event(Event::ElectionError);
-				});
+				},
+			)
+			.map_err(|e| {
+				log::error!(
```

Comment:
would be good if you make a custom `macro_rules! log` for this pallet as well. Also emit a logs upon successful election as well.

Reply:
done!